build/pack.c

```diff
@@ -14,6 +14,8 @@
 #include <rpm/rpmfileutil.h>
 #include <rpm/rpmlog.h>
 
+#include <prtpool.h>			/* NSPR thread pools */
```

Comment:
It'd be pretty hard for me to compile this on Mac OS now, as the Netscape stuff is difficult to bootstrap.